### DISCUSSION TITLE
Yaml anchors

### DISCUSSION THREAD

**Discussion by pmatos** [community/community#25545] [Actions]
<p>What’s the current story with yaml anchors in github actions? Are they supported?</p>
<p>Paulo</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Currently Github doesn’t support YAML anchors, but it is being tracked, it’s recommended to vote on this similar <a href="https://github.community/t/support-for-yaml-anchors/16128">ticket</a>, thanks.</p>

**Reply by merusso** [reply]
That link redirects to the root [GitHub Community Discussions](https://github.com/orgs/community/discussions/) page now. Maybe it was deleted?

Here's a similar feature request: [Support YAML Anchors #4501](https://github.com/orgs/community/discussions/4501)
